.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
if you add this you will only get security update PRs, rather than being spammed by every new version change.
```
    # only allow security updates for dependencies, not version updates  
    # https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file
    open-pull-requests-limit: 0
```

Reply:
I don't mind doing this, but there are only two dependencies in the go.mod file, I think we can afford to have a PR for every update?